Cargo.toml

```diff
@@ -109,7 +112,7 @@ embassy-executor = { git = "https://github.com/kaspar030/embassy", branch = "for
 embassy-hal-internal = { git = "https://github.com/kaspar030/embassy", branch = "for-riot-rs-240605" }
 embassy-nrf = { git = "https://github.com/kaspar030/embassy", branch = "for-riot-rs-240605" }
 embassy-net = { git = "https://github.com/kaspar030/embassy", branch = "for-riot-rs-240605" }
-embassy-rp = { git = "https://github.com/kaspar030/embassy", branch = "for-riot-rs-240605" }
+embassy-rp = { git = "https://github.com/elenaf9/embassy", branch = "for-riot-rs-multicore-240821" }
```

Comment:
The fork (which itself is a fork of `kaspar:for-riot-rs-240605`) disables the `SIO_IRQ_PROC1` interrupt  handler (for FIFO messages between the cores) in `embassy_rp`.
It would conflict with our own handler. They use it to start/resume core 1, we need it to trigger the scheduler on the other core.